gui/package_app

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash -e
```

Comment:
I think the BSD guys prefer "/bin/sh" and "set -e" instead. They probably don't use our packaging.